### DISCUSSION TITLE
[OnlyFans] I can't download content from a specific model.

### DISCUSSION THREAD

**Discussion by ArleneExcalibur** [AAndyProgram/SCrawler#202] [Q&A] [answered]
So I've just setup SCrawler and wanted to try on a model I paid for but couldn't download anything. I always got the message "Some posts didn't download. You can see them in the missing posts form." I thought I had messed up something on the settings maybe. But then I wanted to try another model, a free trial subscription, and it worked flawlessly with this one. I have yet to try on another. Is there a solution to this?

![Captura de pantalla 2024-06-26 112643](https://github.com/AAndyProgram/SCrawler/assets/112824039/b26af101-45fe-444f-906f-b8eeca06462e)
![Captura de pantalla 2024-06-26 112703](https://github.com/AAndyProgram/SCrawler/assets/112824039/22d75607-f7f8-40e7-a349-6548b6f1da64)

**Comment by AAndyProgram** [comment, marked as answer]
https://github.com/AAndyProgram/SCrawler/issues/177#issuecomment-2188343227

----

**Join our Discord server**: https://discord.gg/uFNUXvFFmg
